### PR TITLE
Migrate to PureScript 0.14.0

### PR DESCRIPTION
**Pull request by fsoikin**
This PR makes updates dependency versions to build with the shiny new PureScript 0.14.0 compiler.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -9,15 +9,15 @@
...
   "dependencies": {
-    "purescript-prelude": "^4.0.0",
-    "purescript-aff": "^5.0.0",
-    "purescript-random": "^4.0.0",
-    "purescript-quickcheck": "^5.0.0",
-    "purescript-spec": "^3.1.0"
+    "purescript-prelude": "^5.0.0",
+    "purescript-aff": "^6.0.0",
+    "purescript-random": "^5.0.0",
+    "purescript-quickcheck": "master",
```
Comment (fsoikin) [inline bower.json]: I see this repo has [a "prepare 7.0.0 release" commit](https://github.com/purescript/purescript-quickcheck/commit/7db589bc9a5fb058291863900c5d93c03cc28e5c) with all the right updates, but no actual version tag.

@thomashoneyman @kl0tl

Comment (thomashoneyman) [reply bower.json]: There’s a v7 tag:
https://github.com/purescript/purescript-quickcheck/releases

Comment (fsoikin) [reply bower.json]: Oh, I must have missed that. My bad. Fixed now.

File: bower.json [hunk 2]
```diff
@@ -9,15 +9,15 @@
...
+    "purescript-aff": "^6.0.0",
+    "purescript-random": "^5.0.0",
+    "purescript-quickcheck": "^7.0.0",
+    "purescript-spec": "^5.0.0"
   },
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "git://github.com/owickstrom/purescript-spec-quickcheck.git"
+    "url": "https://github.com/owickstrom/purescript-spec-quickcheck.git"
```
Comment (kl0tl) [inline bower.json]: This library has been relocated to the purescript-spec org in the registry to match its actual location on GitHub:

```suggestion
    "url": "https://github.com/purescript-spec/purescript-spec-quickcheck.git"
```

Comment (fsoikin) [reply bower.json]: Yes, but it still has this URL [on Pursuit](https://pursuit.purescript.org/packages/purescript-spec-quickcheck/3.1.0), so if I change it, `pulp publish` will refuse to publish.

Is there a way to convince it to publish anyway?

Comment (kl0tl) [reply bower.json]: Is the Pursuit URL really relevant? I thought only the registry URL mattered.

Comment (fsoikin) [reply bower.json]: I'm not entirely sure what you call "registry" (package sets?), but in the past, when I tried to publish a package to Pursuit, and the URL didn't match, `pulp publish` would refuse to publish and tell me that the URL doesn't match.

Comment (thomashoneyman) [reply bower.json]: The registry is here:
https://github.com/purescript/registry

Comment (fsoikin) [reply bower.json]: Oh, that. But it's not quite live yet, is it?

Comment (fsoikin) [reply bower.json]: But anyway, if you guys think publishing on Pursuit is not important, I totally defer to your judgement.

Comment (thomashoneyman) [reply bower.json]: It’s the other way around — you should be able to publish with the new url

Comment (fsoikin) [reply bower.json]: I'm not quite sure why that is, but I trust you. 
In any case, I'm not a maintainer here, so I can't publish myself.

Comment (thomashoneyman) [reply bower.json]: You can’t publish a new version, but anyone can still publish to pursuit once it’s out! (I’m not a maintainer either)

Comment (fsoikin) [reply bower.json]: Yes, creating version tag is what I meant.

**Comment by thomashoneyman** [comment]
Hi @owickstrom -- would you mind providing access to @fsoikin, myself, or both for the `spec` organization? That way we can release the other libraries under the organization.

**Comment by owickstrom** [comment]
> Hi @owickstrom -- would you mind providing access to @fsoikin, myself, or both for the `spec` organization? That way we can release the other libraries under the organization.

Absolutely, I missed the org access. Thanks!

**Pull request merged**